cpp/kinematic_icp/3rdparty/bonxai/bonxai.patch

```diff
@@ -0,0 +1,52 @@
+diff --git a/bonxai_core/include/bonxai/bonxai.hpp b/bonxai_core/include/bonxai/bonxai.hpp
+index 7d360cf..e43771e 100644
+--- a/bonxai_core/include/bonxai/bonxai.hpp
++++ b/bonxai_core/include/bonxai/bonxai.hpp
+@@ -367,7 +367,7 @@ inline size_t Grid<DataT>::memUsage() const {
+ template <typename DataT>
+ inline void VoxelGrid<DataT>::releaseUnusedMemory() {
+   std::vector<CoordT> keys_to_delete;
+-  for (const auto& [key, inner_grid] : root_map) {
++  for (auto& [key, inner_grid] : root_map) {
+     for (auto inner_it = inner_grid.mask().beginOn(); inner_it; ++inner_it) {
+       const int32_t inner_index = *inner_it;
+       auto& leaf_grid = inner_grid.cell(inner_index);
+@@ -404,9 +404,9 @@ inline VoxelGrid<DataT>::VoxelGrid(double voxel_size, uint8_t inner_bits, uint8_
+ template <typename DataT>
+ inline CoordT VoxelGrid<DataT>::posToCoord(double x, double y, double z) const {
+   return {
+-      static_cast<int32_t>(std::nearbyint(x * inv_resolution)),
```

Comment:
Have you discussed this with @dfconti? Maybe something we could PR on Bonxai?

Reply:
Already did before opening this PR, see https://github.com/facontidavide/Bonxai/pull/37 ;)